### DISCUSSION TITLE
Awesome Job - Even works on a Pi 2b.

### DISCUSSION THREAD

**Discussion by airshane** [raspicamplayer/camplayer#68] [General]
I was looking for a way to have a security camera view in my home office when it might hard to hear the doorbell, etc, especially if I'm waiting on an import package.  This little app did the trick.  Just an FYI for anyone going down this path, I'm using an older 4K monitor (Dell UP3214Q) with an old pi2b driving the display.  The 2b is only capable of full HD, but it displays fine at a distance and I have 3 streams (one medium, two low quality) being displayed on the monitor.  I might get a cheaper TV for the display, as I have plans for this 4K monitor in the future, but for now it gets the job done for $0.  Great job!

**Comment by Meins321** [comment]
thats the prime function for this player if i am not mistaken..

but you still run it all the time, can you test what happens when you switch off the monitor and switch it on again while the pi stays online all the time...

i do think about energy consumtion from the display might as well be saved if no one looks at it...
